store/auth.ts

```diff
@@ -85,14 +85,27 @@ export default class AuthModule extends VuexModule implements AuthState {
     }
 
     @Action({rawError : true})
-    public async signUpWithEmail({email, password} : {email : string, password: string})
+    public async signUpWithEmail({email, password, name} : {email : string, password: string, name:string})
     {
-        try
-        { return await firebaseAuth.createUserWithEmailAndPassword(email, password);
-        } catch(error)
-        {
-            throw(error)
-        }
+        // try
+        // { return await firebaseAuth.createUserWithEmailAndPassword(email, password)
+        //     // .then((userInfo) =>{ userInfo.user.updateProfile({displayName: name}).then(firebaseAuth.currentUser.reload()).then(() => {console.log(firebase.auth().currentUser.displayName)});
+        // } catch(error)
+        // {
+        //     throw(error)
+        // }
+
+        await firebaseAuth.createUserWithEmailAndPassword(email, password).catch(error => {
+            if (error.code === 'auth/email-already-in-use') {
+                console.log('in use')
```

Comment:
Handle errors better. If this error happens, how will the user know? They aren't going to go looking through the console or whatever. Don't handle this error in the store, you should be handling this error where you are calling the function. You can do 
```
try { ...call the fucntion
} catch (error)
{
this.$vs.notification({message : "awww shucks"}); 
}

``` 
or something to make sure the user knows what went wrong.

Reply:
check 
![image](https://user-images.githubusercontent.com/46302202/90992802-032bd400-e580-11ea-8f44-4630b85eaf23.png)
from `Pages/Suggestions.vue` for an example of how to notify users on error